components/builder-originsrv/src/migrations/2018-03-05-224023_origin_secrets/down.sql

```diff
@@ -1 +0,0 @@
--- This file should undo anything in `up.sql`
```

Comment:
You might want to add a couple lines in the Migrations doc that we should leave the down.sqls blank (that will also close another issue we currently have open :)